r/README.Rmd

```diff
@@ -36,17 +36,26 @@ Conda users on Linux and macOS can install `arrow` from conda-forge with
 conda install -c conda-forge r-arrow
 ```
 
-On macOS and Windows, installing a binary package from CRAN will handle Arrow's C++ dependencies for you. On Linux, unless you use `conda` you'll need to first install the C++ library. See the [Arrow project installation page](https://arrow.apache.org/install/) to find pre-compiled binary packages for some common Linux distributions, including Debian, Ubuntu, and CentOS. You'll need to install `libparquet-dev` on Debian and Ubuntu, or `parquet-devel` on CentOS. This will also automatically install the Arrow C++ library as a dependency. Other Linux distributions must install the C++ library from source.
+On macOS and Windows, installing a binary package from CRAN will handle
+Arrow's C++ dependencies for you. On Linux, unless you use `conda`,
+the R package will have to compile its bindings from source
+and it will need to find or download the C++ dependencies.
+As of the 0.16.0 release, this dependency resolution is automatic on most
```

Comment:
Are we settled on the 0.16 release?

Reply:
IIUC; but also if we do 1.0, this will still be true enough (since 1.0 > 0.16).